CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - install `redmine_extended_rest_api` plugin during docker build (#47)
 - update deprecated bundle install call (#34)
+- Upgrade base image to 3.12.4-1
```

Comment:
I don't know if the base image is publicly available, but if it is one could link it.

Reply:
It is publicly available :)